diff -up spacebar-21.12/CMakeLists.txt.omv~ spacebar-21.12/CMakeLists.txt
--- spacebar-21.12/CMakeLists.txt.omv~	2022-01-07 03:27:23.502283412 +0100
+++ spacebar-21.12/CMakeLists.txt	2022-01-07 03:27:31.144308872 +0100
@@ -34,7 +34,7 @@ ecm_setup_version(${PROJECT_VERSION}
 find_package(Qt5 ${QT_MIN_VERSION} REQUIRED COMPONENTS Core Gui Network Qml Quick QuickControls2 Widgets Sql Xml)
 find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS Kirigami2 I18n People Notifications Config CoreAddons DBusAddons ModemManagerQt)
 find_package(PhoneNumber COMPONENTS PhoneNumber REQUIRED)
-find_package(QCoro REQUIRED)
+find_package(QCoro5 REQUIRED COMPONENTS Core DBus)
 
 qcoro_enable_coroutines()
 
diff -up spacebar-21.12/src/CMakeLists.txt.omv~ spacebar-21.12/src/CMakeLists.txt
--- spacebar-21.12/src/CMakeLists.txt.omv~	2022-01-07 03:27:34.653320560 +0100
+++ spacebar-21.12/src/CMakeLists.txt	2022-01-07 03:27:45.267355909 +0100
@@ -39,7 +39,7 @@ target_link_libraries(spacebar
     kTelephonySupport
     macros
     mms
-    QCoro::QCoroCore
-    QCoro::QCoroDBus
+    QCoro::Core
+    QCoro::DBus
 )
 install(TARGETS spacebar ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
